Library/Homebrew/cmd/cleanup.rb

```diff
@@ -62,6 +62,10 @@ def cleanup_cache
         cleanup_path(path) { path.unlink }
         next
       end
+      if path.to_s.include?("java_cache")
```

Comment:
The code of the `clean` command is probably not the best example to learn from. :wink: I'm very much in favor of doing the right and safe thing, especially if it is no big deal, instead of wiping unrelated stuff that happens to have the “wrong” file name (even if the effect is limited to the cache directory).